packages/stark-build/config/index-html.transform.js

```diff
@@ -46,10 +52,9 @@ function replacePlaceholdersByValues(indexHtml) {
 	return indexHtml.replace(regex, getRealValue);
 }
 
-module.exports = (targetOptions, indexHtml) => {
+module.exports = function transformIndexHtml(targetOptions, indexHtml, commonDataToOverride) {
```

Comment:
Could we keep it more and less as before?


```suggestion
module.exports = (targetOptions, indexHtml, commonDataToOverride) => {
```

Reply:
recommended by sonnarcloud